server/boards/routes.ts

```diff
@@ -759,24 +762,21 @@ router.get(
   ensureBoardAccess, ensureLoggedIn,
   ensureBoardPermission(BoardPermissions.viewRolesOnBoard),
   async (req, res) => {
-    try {
       const { board_id } = req.params;
       const boardRoles = await getBoardRoles({
         boardExternalId: board_id,
       });
+      if (!boardRoles){
```

Comment:
Since now you aren't returning null, this case will have an exception thrown rather than returning null, so you still need the `try`/`catch` here.

Reply:
oops lol